### ISSUE TITLE
Cant Find a Specific Process in Both version (64/86)! 

### ISSUE THREAD

**Issue by EleaSkipper** [ReClassNET/ReClass.NET#159]
Hello , I'm trying to find some structures of PUBGM/Gameloop but I can't find the Process of Emulator so I can Attach the ReClass.Net to it? [https://i.ibb.co/WBsh89h/dd.jpg](url)

**Comment by GH-Rake** [comment]
If androidemulator.exe is x86 make sure you're using the x86 version of ReClass.NET

**Comment by EleaSkipper** [comment]
yes, I tried it both (64/86) and I still can't find the Emulator.exe in Processes List ,I'm trying to attach reclass.net to the Gameloop (aow_exe.exe)

**Comment by KN4CK3R** [comment]
Which emulator is that?

**Comment by hammad2224** [comment]
I think he is trying on Gameloop.
If you do not run reclass.net as administrator you do not see the gameloop subprocess
so try to run reclass.net as administrator and you will see the child process for gameloop
